db/migrate/20160504074011_set_report_type_from_chargeback_to_chargeback_vm_on_miq_reports.rb

```diff
@@ -0,0 +1,27 @@
+class SetReportTypeFromChargebackToChargebackVmOnMiqReports < ActiveRecord::Migration[5.0]
+  CHARGEBACK_REPORT_DB_MODEL = "Chargeback".freeze # old
+  CHARGEBACK_VM_REPORT_DB_MODEL = "ChargebackVm".freeze # new
+
+  class MiqReport < ActiveRecord::Base
+    serialize :db_options
+  end
+
+  def up
+    say_with_time('Set Chargeback to ChargebackVm on MiqReports for db column rtp_type in db_options') do
+      MiqReport.where(:db => CHARGEBACK_REPORT_DB_MODEL).all.each do |miq_report|
+        miq_report.db_options[:rpt_type] = miq_report.db = CHARGEBACK_VM_REPORT_DB_MODEL
```

Comment:
@lpichler Does `db_options[:rpt_type]` need to be downcased here like it was for "chargeback"?